### PR TITLE
Initial step for Ghost deploy

### PR DESCRIPTION
**Pull request by yrobla**

### CODE REVIEW

File: site/blog/2022/deploy-ghost-ec2-advanced.mdx [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+-- Create recordset
+SELECT * FROM iasql_begin();
+INSERT INTO hosted_zone (domain_name) VALUES ('test-ghost.com.');
+INSERT INTO resource_record_set (name, parent_hosted_zone_id, record_type, record, ttl) 
+select 'blog.test-ghost.com.', id , 'A', '<instance_public_ip>', 300
+from hosted_zone where domain_name='test-ghost.com.';
+SELECT * FROM iasql_commit();
+```
+
+After some period of time, and assuming that the domain is properly pointing to <a href="https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/migrate-dns-domain-in-use.html#migrate-dns-change-name-servers-with-provider">AWS nameservers</a>,
```
Comment (aguillenv) [inline site/blog/2022/deploy-ghost-ec2-advanced.mdx:34]: what do you mean by "... and assuming that the domain is properly pointing to AWS nameservers"?

Comment (yrobla) [reply site/blog/2022/deploy-ghost-ec2-advanced.mdx:34]: Well, that you go into your domain registrar and configure the nameservers properly. That link is explaining how to configure that. Do you think i should explain in another way?

Comment (aguillenv) [reply site/blog/2022/deploy-ghost-ec2-advanced.mdx:31]: I wonder if there's something they just have to make sure it is or if it is something they actually have to do actively. If the later, then yes, I think we should rephrase it to be more actionable

Comment (yrobla) [reply site/blog/2022/deploy-ghost-ec2-advanced.mdx:31]: well yes, they need to do it. When they buy the domain it points to the default nameservers of the registrar. If they need to use AWS Route 53, they need to change those nameservers to point to it.

File: site/blog/2022/deploy-ghost-ec2-advanced.mdx [hunk 2]
```diff
@@ -0,0 +1,187 @@
+---
+slug: deploy-ghost-ec2-advanced
+title: Advanced configuration for deploying Ghost on EC2
+date: 2022-12-28
+authors: [yrobla]
+---
+
+# Advanced configuration for deploying <a href="https://ghost.org/">Ghost</a> on EC2
```
Comment (depombo) [inline site/blog/2022/deploy-ghost-ec2-advanced.mdx:8]: the other post is fairly short. I'm curious, why did you opt to split them up?

Comment (yrobla) [reply site/blog/2022/deploy-ghost-ec2-advanced.mdx:8]: I did it to isolate what was the basic setup vs more advanced config, to avoid a very long/complex post. But if you think it's better i can add this to the initial ghost blogpost.

**Review by aguillenv — APPROVED** [review]

**Review by mtp1376 — APPROVED** [review]

File: site/blog/2022/deploy-ghost-ec2-advanced.mdx [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+In our previous <a href="https://iasql.com/blog/deploy-ghost-ec2/">blogpost</a> we showed how to install <a href="https://ghost.org/">Ghost</a> in an EC2 instance, using a simple configuration. While it is ok
+for testing and development purposes, it was not ready for production. In order to be ready, it needs some essential features, that we will be illustrating on this new post.
```
Comment (dfellis) [inline site/blog/2022/deploy-ghost-ec2-advanced.mdx:11]: Nit: "essential features" feels too vague to me, can we just list out the things we need: domain name, SSL cert, HTTPS-to-HTTP load balancer, and RDS? (Because you could also consider an auto-scaling group for the instance behind the load balancer as something else needed for production, and we aren't covering that here.)

File: site/blog/2022/deploy-ghost-ec2-advanced.mdx [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+
+The current deployment is served as HTTP, something not really suitable for production, where all traffic needs to be served via HTTPS. In order to achieve that, we can request a certificate for the
+domain using <a href="https://aws.amazon.com/certificate-manager">AWS ACM</a> functionality. The certificate will be validated automatically using the <a href="https://docs.aws.amazon.com/acm/latest/userguide/dns-validation.html">DNS method</a>:
+
+```sql TheButton[Request certificate and validate via DNS]="Request certificate and validate via DNS"
+ -- Install modules
+SELECT * FROM iasql_install ('aws_acm');
+
+-- request certificate
+SELECT * FROM certificate_request('test-ghost.com', 'DNS', 'us-east-2', '');
```
Comment (dfellis) [inline site/blog/2022/deploy-ghost-ec2-advanced.mdx:60]: Why the space on the function arguments in one and not the other?

File: site/blog/2022/deploy-ghost-ec2-advanced.mdx [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+```sql TheButton[Request certificate and validate via email]="Request certificate and validate via email"
+ -- Install modules
+SELECT * FROM iasql_install ('aws_acm');
+
+-- request certificate
+SELECT * FROM certificate_request('test-ghost.com', 'EMAIL', 'us-east-2', '');
+```
+
+Once the certificate has been properly issued, it needs to be associated with the current EC2 instance. This is achieved by creating a
+<a href="https://aws.amazon.com/elasticloadbalancing/">Load Balancer</a> - application, internet facing - and properly associating the certificate and domain:
```
Comment (dfellis) [inline site/blog/2022/deploy-ghost-ec2-advanced.mdx:75]: It could be fed into the `nginx` server without involving the load balancer, to be fair. Perhaps we should state that the easiest way to use the certificate is with the AWS-provided load balancer?
